tests/test_load_video_frames.py

```diff
@@ -438,7 +438,7 @@ def test_megadetector_lite_yolox_dog(tmp_path):
         [
             "ffmpeg",
             "-r",
-            "60",
+            "30",
```

Comment:
It seems like something funky was happening in the creation of the video from individual frames. We separately test MegadetectorLite just on the stack of 10 frames and get the right detections out.

https://github.com/drivendataorg/zamba/blob/70e79f2e5cdd866607eaaa24e98a655d44d2e210/tests/test_megadetector_lite_yolox.py#L70-L90

I can't explain why but any frame rate other than 60 makes this test pass 🤷‍♀️ I use 32 here since that's typically what we see in videos coming in